cluster/charts/crossplane/templates/clusterrole.yaml

```diff
@@ -9,6 +9,8 @@ metadata:
     heritage: {{ .Release.Service }}
 aggregationRule:
   clusterRoleSelectors:
+  # TODO(negz): Remove aggregate-to-crossplane-admin. The Crossplane service
+  # account should not be granted the same access as a Crossplane administrator.
```

Comment:
Yeah, I noticed this while working on the RBAC manager but it's otherwise unrelated to this PR. Currently we grant the Crossplane pod all the privileges of the crossplane-admin account, and I'm quite confident that's more than it needs. I don't think there's any urgency to change it though, so it can probably wait until we tidy up the Helm RBAC roles as part of removing support for the deprecated package manager.